theodo/a11yinyerface
# Add final page fail

## What is this pull request doing ?


This PR regroups the creation of the egg page and the final page in case of failure
https://github.com/theodo/a11yinyerface/issues/41 
https://github.com/theodo/a11yinyerface/issues/50
https://github.com/theodo/a11yinyerface/issues/62

## Screenshot




<img width="1407" alt="Capture d’écran 2023-07-04 à 11 31 14" src="https://github.com/theodo/a11yinyerface/assets/73390978/2e98dacb-d579-4496-b59b-13b2477f0e7c">



<img width="1407" alt="Capture d’écran 2023-07-03 à 15 08 26" src="https://github.com/theodo/a11yinyerface/assets/73390978/7e80c5fa-391a-4dfb-931f-4373a128d075">

## How is it done ?



## How to test ?



Go to https://deploy-preview-73--a11yinyerface.netlify.app/egg and add one egg to your cart
Go to https://deploy-preview-73--a11yinyerface.netlify.app/final : you should see the failure page indicating you that you have no flour or apple
Go to https://deploy-preview-73--a11yinyerface.netlify.app/fruits and to https://deploy-preview-73--a11yinyerface.netlify.app/flour, add the right ingredients and then check that the final page is the success one
